protocol/messenger_handler.go

```diff
@@ -3827,7 +3827,7 @@ func (m *Messenger) handleSyncKeypairInternal(state *ReceivedMessageState, messa
 	if m.walletAPI != nil {
 		err := m.walletAPI.SetPairingsJSONFileContent(message.KeycardPairings)
 		if err != nil {
-			return err
+			m.logger.Warn("handleSyncKeypairInternal unable to set pairing.json file", zap.Error(err))
```

Comment:
Presumably this is a non-critical error, we don't need to return it.

Reply:
yeah im not sure about this one, @saledjenic ?

Reply:
I would go with `return` if an error occurs. 
`handleSyncKeypairInternal` is used in 2 scenarios:
- while local pairing (where we use `fromLocalPairing = true`)
- while applying the sync message from the paired device (where we use `fromLocalPairing = false`)

But an interesting thing now is that I don't see part of the code that is calling `func (m *Messenger) HandleSyncKeypair(state *ReceivedMessageState, message *protobuf.SyncKeypair, statusMessage *v1protocol.StatusMessage) error ` function. Seems it got removed at some point. So we should remove that function if we don't need this handling (what I am not sure about).